packages/dd-trace/src/config.js

```diff
@@ -178,7 +178,7 @@ function validateNamingVersion (versionString) {
  * @param {string | string[]} input
  */
 function splitJSONPathRules (input) {
-  if (!input) return null
```

Comment:
does the new lint rule forbid returning null ? i think it's a legit use case.
null !== undefined, technically and semantically

Reply:
No, it does not forbid that. I changed it to be able to use a default argument in the spot that receives the values from here